docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Allows you to conveniently run [Sphinx][1] commands to build docs. Typical usage is:

```bash
cd docs
sphinx-quickstart
make html
## To apply changes made to docs source-files
make clean && make html
```
[1]: https://www.sphinx-doc.org/en/master/usage/quickstart.html